frontend/src/components/ReleaseDescription.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/macro'
+
+export default function ReleaseDescription({release}){
+
+    const renderReleaseYear = () => {
+        if (release.releaseYear === 0){
+            return <></>
+        } else {
+            return <h3>{release.releaseYear}</h3>
+        }
+    }
+
+    return (
+        <Release>
+            <img src={release.coverImageUrl} alt={'Shows the release cover'}/>
+            <section>
+                <h3>{release.fullTitle}</h3>
+                {renderReleaseYear()}
+            </section>
+        </Release>
+    )
+}
+
+
+
+const Release = styled.section`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  margin: 0px 20px 0px 20px;
```

Comment:
```suggestion
  margin: 0px 20px;
```

Reply:
I changed it and it seems fine ... for my understanding: just using two values for margin, the first one specifies top & bottom and the second one specifies right & left ?!

Reply:
correct 👍🏽